assets/js/modules/analytics/util/countries-data.js

```diff
@@ -0,0 +1,3042 @@
+export const countries = {
```

Comment:
This feels like a precarious thing for us to maintain—is there an npm library that'd get updates with the same type of data?

Reply:
I had the same thought. This is essentially a dump from an Analytics API endpoint so it's guaranteed to match. I think we should double check to see if this can be replaced with a module though.

Reply:
Agreed. Maybe we can even request it on demand if it's not available in a module. I'm just wary about this endpoint unexpectedly changing, since it's apparently not documented. Let's follow up on this @adamsilverstein